cmd/core/main.go

```diff
@@ -135,40 +134,28 @@ func main() {
 		setupLog.Error(err, "unable to create manager")
 		os.Exit(1)
 	}
-
 	ns := util.PodNamespace(systemNamespace)
-	storageURL, err := url.Parse(fmt.Sprintf("%s/bundles/", httpExternalAddr))
-	if err != nil {
-		setupLog.Error(err, "unable to parse bundle content server URL")
-		os.Exit(1)
-	}
-
-	localStorage := &storage.LocalDirectory{
-		RootDirectory: provisionerStorageDirectory,
-		URL:           *storageURL,
-	}
-
+	var bundleCA io.Reader
 	var rootCAs *x509.CertPool
 	if bundleCAFile != "" {
+		bundleCA, err = os.Open(bundleCAFile)
```

Comment:
Do we need a defer statement here to ensure we're closing the file descriptor?

Reply:
Yes, I'll add it.